jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/pinning/ConnectionPinningManager.kt

```diff
@@ -98,7 +99,7 @@ class DefaultConnectionPinningManager :
         }
 
         var connectionToPin = if (oldConnection is AwsBearerTokenConnection) oldConnection else newConnection
-        if (computeOnEdt { showDialogIfNeeded(oldConnection, newConnection, featuresString) }) {
+        if (showDialogIfNeeded(oldConnection, newConnection, featuresString)) {
```

Comment:
Should this not be on EDT?

Reply:
made just the part that needed to be on EDT instead of the entire call